client/scss/components/_footer.scss

```diff
@@ -85,12 +85,31 @@
   width: 100%;
 
   @include media-breakpoint-up(sm) {
-    width: 250px;
+    width: theme('spacing.64');
   }
 
   &--primary {
+    // Support basic layout support for items added via `extra_footer_actions`
+
+    align-items: stretch;
+    display: grid;
```

Comment:
As I understand, this allows us to remove the `w-grid` from these:

https://github.com/wagtail/wagtail/blob/0599a56d817cc0a728b85ec936ea1c1d62cf19c9/wagtail/snippets/templates/wagtailsnippets/snippets/create.html#L15

https://github.com/wagtail/wagtail/blob/0599a56d817cc0a728b85ec936ea1c1d62cf19c9/wagtail/snippets/templates/wagtailsnippets/snippets/edit.html#L16

which were added after https://github.com/wagtail/wagtail/pull/11456#discussion_r1455042722.